kakarot_scripts/deployment/declarations.py

```diff
@@ -0,0 +1,16 @@
+from kakarot_scripts.constants import DECLARED_CONTRACTS
+from kakarot_scripts.utils.starknet import declare, dump_declarations
+
+
+async def declare_contracts():
+    class_hash = {contract: await declare(contract) for contract in DECLARED_CONTRACTS}
+    dump_declarations(class_hash)
+
+
+if __name__ == "__main__":
+    from uvloop import run
```

Comment:
for all the sub scripts, put the same final logic (with appropriate naming)

The `xxx_sync` version let us define a uv script and do `uv run xxx`

```
# %% Run
def main_sync():
    run(main())


if __name__ == "__main__":
    main_sync()
```